plugins/ruby-foreman-webhooks/debian/changelog

```diff
@@ -0,0 +1,5 @@
+ruby-foreman-discovery (0.0.1) unstable; urgency=low
```

Comment:
```suggestion
ruby-foreman-webhooks (0.0.1) unstable; urgency=low
```

dpkg fails to properly unpack that source package, and thus you get wild errors nobody can understand :)